cmake/projects/OpenCV/hunter.cmake

```diff
@@ -12,6 +12,19 @@ include(hunter_download)
 include(hunter_pick_scheme)
 
 # List of versions here...
+
+hunter_add_version(
+    PACKAGE_NAME
+    OpenCV
+    VERSION
+    "4.1.1-p0"
+    URL
+	"https://github.com/cpp-pm/opencv/archive/v4.1.1-p0.tar.gz"
```

Comment:
Nit: Can you remove the extra space before the URL?

Reply:
Done with NIT